commands/play.js

```diff
@@ -1,5 +1,7 @@
 import { SlashCommandBuilder } from 'discord.js'
 
+export const userSongs = []
```

Comment:
The bot can be connected to multiple voice channels and play different songs at the same time.
So instead of exporting this, we should try to put it on the queue because each vc has their own queue.

Reply:
Added a new key to the queue object and then used the previous exported array as its value to keep track of the userSongs, the playSong file will notice after the second song is being played this avoids an issue that comes down to playSong being executed first than play.js.